commands/uninstall.js

```diff
@@ -11,19 +11,22 @@ module.exports = {
     if (message.author.id == process.env.id) {
       if (args[0] !== undefined) {
         if (args[1] !== "--with-deps") {
-          const init = require(`../commands/${args[0]}.json`);
-          const commandspath = path.join(__dirname, "..", "commands");
+          const init = require(`../custom/${args[0]}.json`);
+          const commandspath = path.join(__dirname, "..", "custom");
 
-          delete require.cache[require.resolve(`./${init.name}.js`)];
+          delete require.cache[require.resolve(`../custom/${init.name}.js`)]; // this doesn't work for some reason
 
           message.channel.send(
-            "deleting " + init.name + " from commands directory"
+            "deleting " + init.name + " from custom commands directory"
           );
           fs.unlink(`${commandspath}/${init.name}.js`, (err) => {
             if (err) {
               message.channel.send(`error. please check logs for details`);
               console.log(err);
             } else {
+              delete require.cache[
+                require.resolve(`../custom/${args[0]}.json`)
+              ];
```

Comment:
all i had to do was clear the `.json` require cache after uninstalling 🤦. see this is what happens when your coding at 3am and your cognitive thinking skills are all over the place lol